examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -246,6 +283,47 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
         }
       };
     }
+    
+    function byCoordRequest(parameters) {
```

Comment:
Also this function...

Reply:
This function is also supposed to be submitted by heatmap other than this ui framework